docs/demonstration/google-e2e.ipynb

```diff
@@ -261,7 +261,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3.9.4 64-bit ('edfi')",
```

Comment:
QUERY: Are the updates to `display_name` and `vscode` intended?

Reply:
Agreed with the reasoning [here](https://github.com/Ed-Fi-Alliance-OSS/LMS-Toolkit/pull/284#discussion_r1026567842)